app/home/homepage.tsx

```diff
@@ -1,24 +1,15 @@
-"use client";
-
 import Link from "next/link";
-import { Button, Col, Row } from "react-bootstrap";
+import { Button } from "react-bootstrap";
 import { PatreonBunnySvgWithoutLink } from "~app/patron/patreon-info";
-import { Run } from "~src/common/types";
-import { Game } from "~app/games/games.types";
+import React from "react";
+import { Col, Row } from "react-bootstrap";
```

Comment:
This should most likely be reverted to:

```suggestion
import { Button, Col, Row } from "react-bootstrap";
import { PatreonBunnySvgWithoutLink } from "~app/patron/patreon-info";
import React from "react";
```
Could combine the imports from `react-bootstrap`